### DISCUSSION TITLE
Unable to View or Continue API-Initiated Chat in Web UI

### DISCUSSION THREAD

**Discussion by GULERTOLGA** [open-webui/open-webui#6279] [Q&A]
Hello,

I've encountered an issue where conversations started via the http://localhost:3000/api/chat/completions API endpoint do not appear in the chat history on the Web UI. This prevents me from continuing the conversation through the interface.

Is there any way to link API-initiated conversations to the Web UI, or a method to transfer the conversation context to the interface? Any guidance would be appreciated.

Thank you!

**Comment by kaptainkangaroo** [comment]
@GULERTOLGA I'm trying to do the same thing, Did you find a solution?
